server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesTest.java

```diff
@@ -459,4 +466,43 @@ void putQuotaShouldBeAbleToRemoveBothQuota() {
         assertThat(maxQuotaManager.getDomainMaxMessage(TROUVÉ_COM)).isEmpty();
     }
 
+    @Test
+    void putQuotaShouldBeAbleToRemoveCountQuota() {
+        given()
+            .body("{\"count\":52,\"size\":42}")
+            .put(QUOTA_DOMAINS + "/" + TROUVÉ_COM.name())
```

Comment:
Not related to the work of this PR but should we really keep constants with french accents around in the code?

Reply:
@blackheaven had a similar question when working on the quota.